src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToControlGeneratedAssemblyInfo.cs

```diff
@@ -418,7 +418,7 @@ public void TestPreviewFeatures(bool enablePreviewFeatures)
             else
             {
                 Assert.True(contains);
-                Assert.True(langVersion.Equals("Preview", System.StringComparison.Ordinal));
+                Assert.True(langVersion.Equals("10.0", System.StringComparison.Ordinal), $"Expected '10.0' but got '{langVersion}'");
```

Comment:
I do not know whether or not it is important to continue using ordinal comparison. After tinkering around a bit it seems like it would be best to do something like

`Assert.Equal("10.0", langVersion, StringComparer.Ordinal);`

Assert.True without a message should probably be avoided, unless the test is on a simple boolean property or something, since it makes it difficult to figure out what the "actual" value was.